compatibility/common/build.gradle.kts

```diff
@@ -77,3 +79,11 @@ tasks.withType<Test>().configureEach {
     excludeTestsMatching("TestNessieCompatibilityExtensions\$*")
   }
 }
+
+// Compatibility tests fail on macOS with a SIGSEGV via rocksdb:
```

Comment:
shouldnt we just skip the rocksdb tests on macos?
or do the other tests provide no value in terms of OS test coverage (i.e. in-memory)?